README.md

```diff
@@ -11,13 +12,17 @@ If you are looking to deploy released versions of the IBM Spectrum Scale CSI Ope
   * OpenShift - Under Operator -> OperatorHub in the OpenShift Console, search 'IBM Spectrum Scale CSI" and install
   * Kubernetes -  See the project on [OperatorHub.io - ibm-spectrum-scale-csi-operator](https://operatorhub.io/operator/ibm-spectrum-scale-csi-operator)
 
+## Support
+
+IBM Spectrum Scale CSI driver is part of the IBM Spectrum Scale offering. Please follow the [IBM support procedure](https://www.ibm.com/mysupport/s/) for any issues with the driver.
+
 ## Report Bugs 
 
 To file issues, suggestions, new features, etc., Open an [Issue](https://github.com/IBM/ibm-spectrum-scale-csi/issues).
 
-## Getting Started and Documentation 
+## Developer Getting Started 
 
-To get started, see our [Documentation](https://ibm-spectrum-scale-csi.rtfd.io/)
+To get started, see our [Developer Documentation](https://ibm-spectrum-scale-csi.rtfd.io/)
```

Comment:
Do we want to keep rtfd doc link even for developer?

Reply:
Keeping Read the Doc for Developers should be fine. I don't think there's a good rationale to have development docs in  KC, because it's an unnecessary barrier for entry to track development  process. @whowutwut and  I have been using the  docs director to preserve more in depth information about the sub components, and testing process  recommendations. Most general users should be grabbed  by the user guide at the top of the README.

Reply:
https://ibm-spectrum-scale-csi.readthedocs.io/en/latest/developers/deployment.html needs update.

Reply:
Please remove the RTD link. Build instructions should be on GitHub, rest everything is covered in KC. 
Lets not hold on to this PR to get the build instructions on GitHub.